.s2i/httpd-cfg/01-commercial.conf

```
@@ -392,6 +392,17 @@ AddType text/vtt                            vtt
     RewriteRule ^pipelines/(1\.10|1\.11|1\.12|1\.13|1\.14|1\.15|1\.16|1\.17|1\.18)/?$ /pipelines/$1/about/understanding-openshift-pipelines.html [L,R=302]
 
 
+
+    # Lightspeed handling unversioned and latest links
+    RewriteRule ^lightspeed/?$ /lightspeed/latest [R=302]
+    RewriteRule ^lightspeed/latest/?(.*)$ /pipelines/1\.0tp1/$1 [NE,R=302]
+
+    # redirect top-level without filespec to the about file
+    RewriteRule ^lightspeed/1.\0tp1/?$ /lightspeed/$1/about/ols-about-openshift-lightspeed.html [L,R=302]
```

```suggestion
    RewriteRule ^lightspeed/(1\.0tp1)/?$ /lightspeed/$1/about/ols-about-openshift-lightspeed.html [L,R=302]
```

I think we need a capture group here to replace it with $1